products/zoningtaxlots/models/product/qa/qa_bbl.sql

```diff
@@ -0,0 +1,29 @@
+{{ config(
+    materialized = 'table'
+) }}
+
+WITH new_version AS (
+    SELECT * FROM {{ ref('int__zoningtaxlots') }}
+),
+
+prev_version AS (
+    SELECT * FROM {{ source('recipe_sources', 'dcp_zoningtaxlots') }}
+),
+
+bbl AS (
+    SELECT
+        sum(CASE WHEN bblnew IS null THEN 1 ELSE 0 END) AS removed,
+        sum(CASE WHEN bblold IS null THEN 1 ELSE 0 END) AS added,
+        'new_version' AS version,
```

Comment:
For these, we should use the env vars that we set in my comment below. See this guide for using [env vars](https://docs.getdbt.com/docs/build/environment-variables) within dbt

Reply:
For this query, let's also simplify it - get rid of the nested query and do this in a single select/join instead